config/sync/core.extension.yml

```diff
@@ -33,6 +33,7 @@ module:
   dpl_breadcrumb: 0
   dpl_cache_settings: 0
   dpl_campaign: 0
+  dpl_cookies: 0
```

Comment:
Denne bør udskiftes med en update hook i `dpl_update`, se fx. https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/1077/commits/1ec893443d56749feefc6a8095d67d9469fec909

Reply:
I have added both the enabling to both the install and update hook now in dpl_update. It seems the process has been to keep the enabling of modules in core.extension.yml as well, even though it is now ignored. I have therefore kept it.